apps/web/src/components/Menu/config/config.ts

```diff
@@ -41,8 +42,13 @@ const config: (
   isDark: boolean,
   languageCode?: string,
   chainId?: number,
-) => ConfigMenuItemsType[] = (t, isDark, languageCode, chainId) =>
-  [
+) => ConfigMenuItemsType[] = (t, isDark, languageCode, chainId) => {
+  const perpChain = chainId === ChainId.ETHEREUM ? 'ethereum' : 'bnbchain'
+  const perpetualUrl = `https://perp.pancakeswap.finance/${perpLangMap(languageCode)}/futures/BTCUSDT?theme=${perpTheme(
```

Comment:
Can we move this part into some common util file so we don't need to write the same code in different components?

Reply:
updated